spec/views/events/schedule.html.erb_spec.rb

```diff
@@ -2,23 +2,52 @@
 
 RSpec.describe 'events/schedule', type: :view do
 
-  before :each do
-    @league = FactoryBot.create(:league, :with_matches,
-                                deadline: Date.parse('23.12.2017'),
-                                startdate: Date.parse('24.12.2017'),
-                                enddate: Date.parse('31.12.2017'),
-                                gameday_duration: 7)
-    assign(:event, @league)
-    assign(:schedule_type, 'league')
-    assign(:matches, @league.matches)
-  end
+  describe 'page layout' do
+    before :each do
+      @league = FactoryBot.create(:league, :with_matches,
+                                  deadline: Date.parse('23.12.2017'),
+                                  startdate: Date.parse('24.12.2017'),
+                                  enddate: Date.parse('31.12.2017'),
```

Comment:
I think you can use the `has_dates` trait here.

Reply:
A test refers to those exact dates (for rendering purposes). I think using the has_dates trait would make things more complicated there.

Reply:
Since these specific dates are important for the test. I'd rather specify them here. If we don't do that and somebody changes the dates in `:has_dates` to someting else. This test will break.
So my point is: It's not only important that our league has dates but it's also important what dates it actually has.